.github/workflows/e2e_tests.yml

```
@@ -1374,3 +1374,163 @@ jobs:
           EMBEDDING_API_VERSION: ${{ secrets.EMBEDDING_API_VERSION }}
         run: uv run python ./cognee/tests/api/test_backend_auth.py
+
+  backwards-compatibility-sqlite:
+    name: Backwards compatibility — SQLite + LanceDB + Kuzu
+    runs-on: ubuntu-22.04
+
+    env:
+      COGNEE_COMPATIBILITY_TEST_VERSION: v0.5.6
+
+      ENV: dev
+      LLM_PROVIDER: ${{ secrets.LLM_PROVIDER }}
```

actionlint (1.7.12)

[error] property "llm_provider" is not defined in object type {actions_runner_debug: string; actions_step_debug: string; aws_access_key_id: string; aws_secret_access_key: string; embedding_api_key: string; embedding_api_version: string; embedding_endpoint: string; embedding_model: string; github_token: string; llm_api_key: string; llm_api_version: string; llm_endpoint: string; llm_model: string; openai_api_key: string}

(expression)